MIP106/MIP106.md

```diff
@@ -352,20 +320,6 @@ The budget available to fund Governance Process Support tasks is:
 
 The DAO Toolkit is a unified governance software system that is used to simplify, organize and standardize all of the governance processes and information of Maker Governance and the Scope Artifacts. Each Scope must, over time, be implemented using the DAO Toolkit as it is developed to make them interactive and make it easier for newcomers to understand how Maker Governance works.
 
-### 3.1: DAO Toolkit Core Development
-
-The Support Facilitators must fund the development of the core underlying software and modules of the DAO Toolkit that is used to implement the Scope Frameworks as software. DAO Toolkit Core Development is funded through the budget defined in *3.1.1B*. This Active Element is modified using the standard AVC process.
-
-#### 3.1.1B
-
-¤¤¤
-
-The budget available to fund DAO Toolkit Core Development tasks is:
-
-* 0 Dai per quarter
-
-¤¤¤
-
 ### 3.2: Standardized DAO Toolkit Patterns
 
 The DAO Toolkit must be developed and used across all of the Scope Framework to unify the user experience of all aspects of Maker Governance as much as possible.
```

Comment:
Again, this is for the section below, as I cannot comment on sections which are not to be modified - Point 4.1.4 makes reference to section 10.1.2, when it should be 4.1.2.

Reply:
That's odd. Nice catch!